### DISCUSSION TITLE
Custom screen layout with Prometheus metrics & WAN from API

### DISCUSSION THREAD

**Discussion by ferni7** [mathoudebine/turing-smart-screen-python#118] [Themes]
https://github.com/ferni7/turing-smart-screen-custom

And a docker image being built in the pipeline - we could add this in here if you're interested @mathoudebine

**Comment by mathoudebine** [comment]
Hi @ferni7
Thanks for sharing your work with the community!
If you want, i can add a link into the Readme of this project, maybe in a new "Other projects" section?
I will have a look to see what could potentially be integrated here 👍 

Please note though that since you integrate this project (https://github.com/mathoudebine/turing-smart-screen-python) into your repository that you now share publicly on Github, you must comply to the terms of the "GNU General Public License v3.0" open-source copyleft license as described here: https://github.com/mathoudebine/turing-smart-screen-python/blob/main/LICENSE notably:
- a copy of the license must be included with the licensed material (so this file https://github.com/mathoudebine/turing-smart-screen-python/blob/main/LICENSE must be included in the `library` folder of your repository)
- your repository must also be licensed under GPLv3: incorporating GPL 3 code into a larger work that is then licensed under Apache 2.0 [is not permitted](https://www.apache.org/licenses/GPL-compatibility.html)

Thanks in advance!

**Reply by ferni7** [reply]
Apologies - Just updated the license to GNU General Public License v3.0 :)

PS I would have preferred to just pull your libraries from pip, any plans to add them there?

**Reply by mathoudebine** [reply]
Thanks for your fast response!
This is a good idea to provide a pip package in the future, this will be implemented in #119
